### PR TITLE
Ground scripts

### PR DESCRIPTION
**Pull request by mateon1**

### CODE REVIEW

File: scripts/__pycache__/scripts.pypy310.pyc
Comment (SethBarberee) [inline scripts/__pycache__/scripts.pypy310.pyc:1]: Accidentally included?

File: scripts/__pycache__/scriptvars.pypy310.pyc
Comment (SethBarberee) [inline scripts/__pycache__/scriptvars.pypy310.pyc:1]: Ditto on all pycache ones

File: include/personality_test1.h [hunk 1]
```diff
@@ -9,7 +9,7 @@ typedef struct PersonalityQuestion
 {
     /* 0x0 */ const u8 *question;
     /* 0x4 */ const MenuItem *answers;
-    /* 0x8 */ const u8 *effects;
+    /* 0x8 */ const u8 (*effects)[16];
```
Comment (SethBarberee) [inline include/personality_test1.h:12]: Maybe a good idea to bind this to `NUM_PERSONALITIES + 3` or modify the enum of personalities so we can bound this with a constant and allow hackers to increase/decrease these later.

File: src/personality_test1.c [hunk 2]
```diff
@@ -215,7 +215,7 @@ static void UpdateNatureTotals(void)
 {
     s32 answerIndex;
     s32 natureIndex;
-    const u8 *pointArray;
+    const u8 (*pointArray)[16];
```
Comment (SethBarberee) [inline src/personality_test1.c:218]: Same comment as the include file with bounding this with an enum value.

**Review by SethBarberee — CHANGES_REQUESTED** [review]

**Review by SethBarberee — APPROVED** [review]

File: src/data/ground/ground_data_a01p01_station.h [hunk 3]
```diff
@@ -0,0 +1,1488 @@
...
+    { 0x54, 0x00,  0x0006,  0x00000000,  0x00000000, NULL },
+    { 0x2d, 0x07,  0x0000,  0x00000000,  0x00000000, NULL },
+    { 0xdf, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(60),
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("......") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("............") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("..................") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Where...") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Where am I?") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Maybe...#W Is this a dream...?") },
```
Comment (AnonymousRandomPerson) [inline src/data/ground/ground_data_a01p01_station.h:45]: `#W` is mapped to the macro `WAIT_PRESS`. Can we use this macro in all these strings?

File: src/data/ground/ground_data_a01p01_station.h [hunk 4]
```diff
@@ -0,0 +1,1488 @@
...
+    { 0x54, 0x00,  0x0002,  0x00000000,  0x00000000, NULL },
+    { 0xdf, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(60),
+    { 0x91, 0x0a,  0x0001,  0x00000000,  0x00000000, NULL },
+    WAIT(30),
+    { 0x34, 0x00, -0x0001,  0x00000000,  0x00000000, _(" I've been waiting for you.") },
+    { 0x30, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(10),
+    { 0x2e, 0x02,  0x0000,  0x0000000c,  0x00000000, NULL },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Ga...#W$n2!") },
```
Comment (AnonymousRandomPerson) [inline src/data/ground/ground_data_a01p01_station.h:403]: `$n2` is a placeholder for a nickname, which is represented by the macro `ARG_NICKNAME_2`. Is it possible to use these argument macros throughout the scripts?

**Comment by mateon1** [comment]
Honestly, those charmap macros read worse to me than the format codes (with exception of color). But yeah, I can do that. Will have to wait until friday/saturday since I will be away from any decent computers for a while.

**Comment by mateon1** [comment]
Managed to do it before I had to go out, everything should be addressed

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**